src/common/attachments/attachmentsTable.component.tsx

```diff
@@ -93,6 +98,7 @@ function AttachmentsTable(props: AttachmentTableProps) {
         size: 350,
         enableGrouping: false,
         Cell: ({ row }) =>
+          row.original.created_time &&
           formatDateTimeStrings(row.original.created_time, true),
```

Comment:
```suggestion
          formatDateTimeStrings(row.original.created_time, true),
```
This is a mandatory field so it doesn't need the conditional

Reply:
i was having issues when i was uploading my own attachments without using mock data, the attachments table would fail to load and the page would 404 because it couldnt find a created or modified time for my uploaded attachment

Reply:
fixed now i just had to rebuild my object storage docker image 😆